imports/api/topics/votings/methods.js

```diff
@@ -53,12 +54,13 @@ export const castVote = new ValidatedMethod({
     if (Meteor.isServer) {
       const updatedTopic = Topics.findOne(topicId);
       updatedTopic.voteEvaluate(false); // writes only voteParticipation, no results
+      if (res === 1) sendHasVotedNoti(_voters, topicId, this.userId);
     }
   },
 });
 
 function closeVoteFulfill(topicId) {
-  const res = Topics.update(topicId, { $set: { closed: true } });
+  const res = Topics.update(topicId, { $set: { closed: true, 'vote.closesAt': new Date() } });
```

Comment:
Itt, mint azt jol eszrevetted, es irtad is a trello kartyan, UTC-s date-et kene letrehozni. Kulonben idozona zavar lesz, es nem ejfelkor zarul le a szavazas, hanem masnap ejjel 2-kor.
Trello-n irtam, hogy kell. Szerintem mostantol csinaljuk ugy.